assets/js/os-style.js

```diff
@@ -0,0 +1,13 @@
+function addOperatingsystemStyle() {
+  var os = 'other';
```

Comment:
Can you put that into a function getOperatingSystem() returning the OS? This makes it easier to have a redirect on jamulus.io/upgrade or jamulus.io/download if we add it one day.

Reply:
Done, although it'll need rethinking/moving in that case anyway.